internal/proto/internal.proto

```diff
@@ -117,6 +117,15 @@ message SearchResults {
   bytes sliced_blob = 10;
   int64 sliced_num_count = 11;
   int64 sliced_offset = 12;
+
+  // request cost
+  RequestCostAggregation costAggregation = 13;
```

Comment:
RequestCostAggregation may be easily misunderstood as a type of request at first glance. It would be better to change the name to avoid confusion.